app/controllers/open_library_controller.rb

```diff
@@ -0,0 +1,9 @@
+class OpenLibraryController < ApiController
+  include AsyncRequest::ApplicationHelper
+  respond_to :json
+
+  def book_info
+    id = execute_async(OpenLibraryService, params[:bibkeys])
+    render json: { id: id, url: async_request.job_url(id) }, status: 202
```

Comment:
Here you can use the status code symbol instead of the number 202.

Reply:
Done